sql/src/main/java/org/vertexium/sql/SqlTableEdge.java

```diff
@@ -39,6 +41,7 @@ InMemoryTableEdge asInMemoryTableElement() {
 
     private static class InMemoryAdapter extends InMemoryTableEdge
             implements Storable<SqlTableElement<InMemoryEdge>, SqlGraph> {
+        private static final long serialVersionUID = SqlTableEdge.serialVersionUID;
```

Comment:
Does this class ever get serialized?  Is it possible to throw an exception instead when it tries to get serialized? 

On December 14, 2015 6:51:34 PM EST, Steven Farley <notifications@github.com> wrote:
>> @@ -39,6 +41,7 @@ InMemoryTableEdge asInMemoryTableElement() {
>>  
>>      private static class InMemoryAdapter extends InMemoryTableEdge
>>              implements Storable<SqlTableElement<InMemoryEdge>,
>SqlGraph> {
>> +        private static final long serialVersionUID =
>SqlTableEdge.serialVersionUID;
>
>What's the reason for this `serialVersionUID` to be the same as its
>outer class?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/v5analytics/vertexium/pull/52/files#r47578126

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.